script.js

```diff
@@ -107,3 +109,70 @@ form.addEventListener('submit', (event) => {
     emailErrorMsg.innerText = 'Please enter the e-mail in lowercase';
   }
 });
+
+const inputData = {};
+const storage = window.localStorage;
+
+function storageAvailable(type) {
+  let storage;
+  try {
+    storage = window[type];
+    const x = '__storage_test__';
+    storage.setItem(x, x);
+    storage.removeItem(x);
+    return true;
+  } catch (e) {
+    return e instanceof DOMException && (
+      e.code === 22
+      || e.code === 1014
+      || e.name === 'QuotaExceededError'
+      || e.name === 'NS_ERROR_DOM_QUOTA_REACHED') && (storage && storage.length !== 0);
+  }
+}
+
+function retrieveFormData() {
+  if (storageAvailable('localStorage')) {
+    const formDataString = storage.getItem('formData');
+    const formData = JSON.parse(formDataString);
+    return formData;
+  }
+  return false;
+}
+
+function populateForm() {
+  const formData = retrieveFormData();
+  if (formData.name) {
+    formName.value = formData.name;
+  }
+  if (formData.email) {
+    email.value = formData.email;
+  }
+  if (formData.message) {
+    formMessage.value = formData.message;
+  }
```

Comment:
- [ ] Kindly fix this line as it is returning null when filling the form for the first time that is causing data to not be stored in the local storage
<img width="824" alt="Screen Shot 2022-04-07 at 15 37 39" src="https://user-images.githubusercontent.com/28307960/162212344-ffa1b058-6ef8-4317-b268-9b504e1af33a.png">